dnf-repoconfig-daemon/polkit/org.rpm.dnf.v0.policy

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1.0/policyconfig.dtd">
+<policyconfig>
+  <vendor>Dnf</vendor>
+  <vendor_url>http://example.com</vendor_url>
+
+  <!-- Action id must be lower case -->
```

Comment:
Indeed all actions on my system are lower-case, but that just seems to be a convention. polkit(8) says `[A-Z][a-z][0-9].-`. Please drop the comment, and decide whether you want to turn "RepoConf" into "repoconf" to actually make it lower-case. :wink: